infra/modules/service/main.tf

```diff
@@ -81,20 +84,23 @@ resource "aws_lb_listener_rule" "alb_http_forward" {
 resource "aws_lb_target_group" "alb_target_group" {
   # you must use a prefix, to facilitate successful tg changes
   name_prefix          = "tg-"
-  port                 = "8080"
+  port                 = var.container_port
   protocol             = "HTTP"
   vpc_id               = var.vpc_id
   target_type          = "ip"
   deregistration_delay = "30"
 
-  health_check {
-    path                = "/health"
-    port                = 8080
-    healthy_threshold   = 2
-    unhealthy_threshold = 10
-    interval            = 30
-    timeout             = 29
-    matcher             = "200-299"
+  dynamic "health_check" {
+    for_each = var.enable_healthcheck ? [0] : []
+    content {
+      path                = "/${local.healthcheck_path}"
```

Comment:
but you just took this `/` off!

Reply:
I wanted to avoid `//healthcheck` and also avoid having to remember whether or not `var.healthcheck_path` needed to have a leading slash or not.